storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3479,9 +3473,10 @@ Rdb_open_tables_map::get_table_handler(const char *const table_name) {
             &m_hash, reinterpret_cast<const uchar *>(table_name), length)))) {
     // Since we did not find it in the hash map, attempt to create and add it
     // to the hash map.
+    // TODO: instrument for PFS
     if (!(table_handler = reinterpret_cast<Rdb_table_handler *>(my_multi_malloc(
```

Comment:
Here and in all other my_ memory allocators - why reinterpret_cast instead of static_cast?